time-tracker-webapi/src/TimeTracker.Library/DeleteInterpreter.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using TimeTracker.Library.Models;
+
+namespace TimeTracker.Library
+{
+
+    public class DeleteInterpretedCommandDto : CommandDtoBase
```

Comment:
we could remove the Dto - was kinda of a carryover from when first started project